packages/arb-token-bridge-ui/src/util/TokenUtils.ts

```diff
@@ -277,12 +278,11 @@ export function sanitizeTokenName(
   }
 
   const isArbitrumOne = isNetwork(options.chain.id).isArbitrumOne
-  const isMainnet = isNetwork(options.chain.id).isMainnet
 
-  // only special case for USDC is Arbitrum One / Mainnet
   if (isTokenMainnetUSDC(options.erc20L1Address)) {
+    // It should be `USD Coin` on all chains except Arbitrum One
     if (isArbitrumOne) return 'Bridged USDC'
-    if (isMainnet) return 'USD Coin'
+    return 'USD Coin'
```

Comment:
now that i think of it, we could skip this return as the `tokenName` should be `USD Coin` from token list anyway

Reply:
Yeah, but then we might to change the logic in https://github.com/OffchainLabs/arbitrum-token-bridge/blob/master/packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearchUtils.ts#L62 to account for that, which is much more work.

Reply:
actually, or if it's `Bridged USDC` on token list, we can return `tokenName` on `isArbitrumOne`
but i guess this way we rely on the token list less
i'd prefer to handle the weird cases on token list though

Reply:
There is nothing wrong with the token list. It shows both L1 and L2 representations, as it should: https://tokenlist.arbitrum.io/ArbTokenLists/arbed_arb_whitelist_era.json

It's up to the UI to handle them properly.

Reply:
ok i see what you mean